source/_patterns/05-pages/autospacing/_autospacing.scss

```diff
@@ -0,0 +1,23 @@
+section.has-background-color--transparent + section.has-background-color--transparent,
+section.has-background-color--light + section.has-background-color--light,
+section.has-background-color--dark + section.has-background-color--dark,
+section.has-background-color--transparent + section.has-background-color--transparent,
+section.has-background-color--light--primary + section.has-background-color--light--primary,
+section.has-background-color--light--secondary + section.has-background-color--light--secondary,
+section.has-background-color--dark--primary + section.has-background-color--dark--primary,
+section.has-background-color--dark--secondary + section.has-background-color--dark--secondary {
+  margin-top: units(-6);
+}
+
+[class*="--has-background-color--transparent"] + [class*="--has-background-color--transparent"],
+[class*="--has-background-color--light"] + [class*="--has-background-color--light"],
+[class*="--has-background-color--dark"] + [class*="--has-background-color--dark"],
+[class*="--has-background-color--light--primary"]
+  + [class*="--has-background-color--light--primary"],
+[class*="--has-background-color--light--secondary"]
+  + [class*="--has-background-color--light--secondary"],
+[class*="--has-background-color--dark--primary"] + [class*="--has-background-color--dark--primary"],
+[class*="--has-background-color--dark--secondary"]
+  + [class*="--has-background-color--dark--secondary"] {
+  padding-top: units(0);
+}
```

Comment:
Could we simplify all these to something like this, or do they need to be explicit?

`[class*="--has-background-color--"] + [class*="--has-background-color--"]`

Reply:
Sorry, nevermind! My brain is not fully working yet this week. :P